### ISSUE TITLE
Math

### ISSUE THREAD

**Issue by Znlsoc** [Znlsoc/LoggingForStudy#4]

**Comment by Znlsoc** [comment]
### 偏导数

**Comment by Znlsoc** [comment]
### 拓扑排序
拓扑排序是对有向无环图的顶点的一种排序，它使得如果存在一条从顶点A到顶点B的路径，那么在排序中B出现在A的后面。

**Comment by Znlsoc** [comment]
### 范数

**Comment by Znlsoc** [comment]
### 定点数
计算机中采用的一种数的表示方法。参与运算的数的小数点位置固定不变。

**Comment by Znlsoc** [comment]
### 卷积

**Comment by Znlsoc** [comment]
### Softmax
在数学，尤其是概率论和相关领域中，Softmax函数，或称归一化指数函数，是逻辑函数的一种推广。
它能将一个含任意实数的K维的向量  z 的“压缩”到另一个K维实向量  z‘ 中，使得每一个元素的范围都在  (0,1)之间，并且所有元素的和为1。

**Comment by Znlsoc** [comment]
## BLAS
摘自维基百科
>BLAS（Basic Linear Algebra Subprograms，基础线性代数程序集）是一个应用程序接口（API）标准，用以规范发布基础线性代数操作的数值库（如矢量或矩阵乘法）。该程序集最初发布于1979年，并用于创建更大的数值程序包（如LAPACK）。在高性能计算领域，BLAS被广泛使用。例如，LINPACK的运算成绩则很大程度上取决于BLAS中子程序DGEMM的表现。
为提高性能，各软硬件厂商则针对其产品对BLAS接口实现进行高度最优化。

#### GEMM: General Matrix to Matrix Multiplication

**Comment by Znlsoc** [comment]
Batchnorm
Conv2d
Eltwise
FC
Pooling 
ReLU
Scale
Softmax
